forJIS7/src/main/java/JavaHomework/Lectures_17.Stream/secondPart/Model/User.java

```diff
@@ -0,0 +1,22 @@
+package JavaHomework.Lectures_17.Stream.secondPart.Model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+
+public class User {
+
+    private Integer id;
+    private String name;
+    private String lastName;
+    private String nickname;
+    private Integer age;
+    private String email;
+    private Integer password;
```

Comment:
Perhaps, the password can include letters. This way, better use String

Reply:
done